### PR TITLE
ncm-openstack: Include ovs and storage support

### LINKED ISSUE #1310: ncm-openstack: add Heat service support
This service is quite useful to setup clusters via yaml files.

https://docs.openstack.org/heat/latest/

### LINKED ISSUE #1315: ncm-openstack: add murano service support
With Heat this is a quite useful service to deploy, install and test new software or services in a cloud environment using an app catalog.

https://docs.openstack.org/murano/latest/reference/key_features.html

### PR DESCRIPTION
**Pull request by alvarosimon**
Requires: `Net::OpenStack::Client version v0.1.3`

* Fix #1310 
* Fix #1315 

This PR includes:
 * Manila support.
 * Cinder support.
 * OVS network support.
 * RPC api support.
 * Heat orchestration support.
 * Murano app catalog support.
 * SSO with openid

### CODE REVIEW

**Comment by alvarosimon** [comment]
Now this PR includes Heat support (fortunately is quite easy to include new OS services in the current component) but it should  be re-re-viewed.
I will test this in our cloud testbed meanwhile.

**Comment by jrha** [comment]
@hpcugentbot please test this please

File: ncm-openstack/src/main/perl/OpenStack/Client.pm [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+=head1 Functions
+
+=over
+
+=cut
+
+use NCM::Component::OpenStack::Logger;
+use NCM::Component::OpenStack::Openrc;
+use Net::OpenStack::Client 0.1.2;
```
Comment (jrha) [inline ncm-openstack/src/main/perl/OpenStack/Client.pm]: We've published 0.1.3 in externals, can we use that version here?

Comment (alvarosimon) [reply ncm-openstack/src/main/perl/OpenStack/Client.pm]: @jrha sure, done!

**Comment by alvarosimon** [comment]
still failing because is not able to find `Net::OpenStack::Client`

**Comment by jrha** [comment]
:disappointed: who can poke Jenkins?

**Comment by stdweird** [comment]
retest this please

**Comment by stdweird** [comment]
test this please

**Pull request merged**